Rubberduck.RegexAssistant/Atom.cs

```diff
@@ -0,0 +1,284 @@
+﻿using Rubberduck.RegexAssistant.i18n;
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Rubberduck.RegexAssistant
+{
+    public interface Atom : IDescribable
```

Comment:
This should be `IAtom` - otherwise `CharacterClass` totally looks like it's derived from a base class. Conventions :smiley: